src/frontend/packages/core/src/shared/components/list/list-types/endpoint/table-cell-endpoint-details/table-cell-endpoint-details.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component, ComponentFactoryResolver, Input, Type, ViewChild, ViewContainerRef } from '@angular/core';
+
+import { EndpointModel } from '../../../../../../../../store/src/types/endpoint.types';
+import { getEndpointType } from '../../../../../../features/endpoints/endpoint-helpers';
+import { TableCellCustom } from '../../../list.types';
+
+@Component({
+  selector: 'app-table-cell-endpoint-details',
+  templateUrl: './table-cell-endpoint-details.component.html',
+  styleUrls: ['./table-cell-endpoint-details.component.scss']
+})
+export class TableCellEndpointDetailsComponent extends TableCellCustom<EndpointModel> {
+
+  @Input() component: Type<TableCellCustom<EndpointModel>>;
+  @ViewChild('target', { read: ViewContainerRef }) target;
+
+  cell: TableCellCustom<EndpointModel>;
+
+  constructor(private componentFactoryResolver: ComponentFactoryResolver) {
+    super();
+  }
+
+  private pRow: EndpointModel;
+  @Input('row')
+  set row(row: EndpointModel) {
+    this.pRow = row;
+
+    const e = getEndpointType(row.cnsi_type);
+    if (e.listDetailsComponent) {
+      if (!this.cell) {
+        const componentFactory = this.componentFactoryResolver.resolveComponentFactory(e.listDetailsComponent);
+        const componentRef = this.target.createComponent(componentFactory);
+        this.cell = componentRef.instance as TableCellCustom<EndpointModel>;
```

Comment:
I think you need a destroy function to clear this custom component up when this component is destroyed.